topostats/entry_point.py

```diff
@@ -345,6 +345,13 @@ def create_parser() -> arg.ArgumentParser:
         required=False,
         help="Whether or not to remove grains that are disconnected from the original mask",
     )
+    process_parser.add_argument(
+        "--unet-confidence",
+        dest="unet_confidence",
+        type=float,
+        required=False,
+        help="Confidence threshold for the UNet model. Smaller is more generous, larger is more strict.",
```

Comment:
Ditto on wording as suggested in `configuration.md` and the line below.

(Very minor and pedantic though so feel free to ignore).

Reply:
```suggestion
        help="Confidence threshold for the UNet model. Smaller is more generous, large is stricter.",
```